public/.htaccess

```diff
@@ -0,0 +1,66 @@
+# Use the front controller as index file. It serves as a fallback solution when
```

Comment:
Haven't seen one of these for like 5 or 6 years.
Have you tried using nginx+fpm? It performs better under higher load and with more requiring apps.

Reply:
Unfortunately I have more experience with apache instead of nginx but yes, you're right ... I should switch to nginx .